javacutil/src/main/java/org/checkerframework/javacutil/TypesUtils.java

```diff
@@ -296,6 +296,17 @@ public static boolean areSamePrimitiveTypes(TypeMirror left, TypeMirror right) {
         return (left.getKind() == right.getKind());
     }
 
+    /**
+     * Returns true iff the arguments are both the same type variables.
+     *
+     * @param v1 a type variable
+     * @param v2 a type variable
+     * @return whether the arguments are the same type variables
+     */
+    public static boolean areSameTypeVariables(TypeVariable v1, TypeVariable v2) {
```

Comment:
This method seems very dangerous... can you add a warning to the method. I think this only works if the TypeVariables are from the same context. If one compares TVs from e.g. a declaration to a TV from a use site, there could be two different `T`s that just happen to have the same name.

Reply:
Thanks for pointing this out! I will leave a comment there.